capsules/src/net/udp/driver.rs

```diff
@@ -6,18 +6,22 @@
 //! Also exposes a list of interface addresses to the application (currently
 //! hard-coded).
 
+/// Syscall number
```

Comment:
This is a bit of a weird comment -- we don't usually document import purposes, particularly not with doc comments

Reply:
I think that is leftover from the file being copied from the 15.4 driver, but it looks like the same doc comment appears in the usb_user driver as well, I will remove all 3.